RUN-4827 - mostly there to parity with v1

tgoc99 opened the pull request:
Kept the old readme in case we want to include that somewhere.

rdepena commented on src/api/interappbus/channel/channels-docs.ts:
```diff
@@ -0,0 +1,352 @@
+    /**
+     * 
+     * Register an error handler. This is called before responding on any error.
+     * @param {function} middleware - Function to be executed in case of an error.\
```
extra \

rdepena commented:
@tgoc99 can you make sure the jsdoc-template changes are PRd into this repo: https://github.com/openfin/jsdoc-template

tgoc99 commented:
I reverted the changes to js-doc template.  All I had done is comment out the events so the ones we added for Window didn't show up.  But per our discussion earlier this week, we haven't decided how we are going to do events so that can fall to this [JIRA](https://appoji.jira.com/browse/RUN-4828) to figure out

licui3936 commented on docs.README.md:
```diff
@@ -1,82 +1,241 @@
+* {@link InterApplicationBus#addSubscribeListener addSubscribeListener} _InterApplicationBus_
+* {@link InterApplicationBus#addUnsubscribeListener addUnsubscribeListener} _InterApplicationBus_
+* {@link Window#animate animate} _Window_
+* [Application](Application.html#Application) _(constructor)_
```
It's not applicable in v2.  Should remove it.

licui3936 commented on docs.README.md:
```diff
@@ -1,82 +1,241 @@
+* {@link ExternalApplication#addEventListener addEventListener} _ExternalApplication_
+* {@link Frame#addEventListener addEventListener} _Frame_
+* {@link System#addEventListener addEventListener} _System_
+* {@link Window#addEventListener addEventListener} _Window_
```
All 'addEventListener ' should be removed in v2.

licui3936 commented on docs.README.md:
```diff
@@ -1,82 +1,241 @@
+* {@link Clipboard#write write} _Clipboard_
+* {@link Clipboard#writeHtml writeHtml} _Clipboard_
+* {@link Clipboard#writeRtf writeRtf} _Clipboard_
+* {@link Clipboard#writeText writeText} _Clipboard_
```
all globalHotkey functions missing

licui3936 commented on docs.README.md:
```diff
@@ -1,82 +1,241 @@
+* {@link Application#getManifest getManifest} _Application_
+* {@link System#getMonitorInfo getMonitorInfo} _System_
+* {@link System#getMousePosition getMousePosition} _System_
+* {@link Window#getNativeWindow getNativeWindow} _Window_
```
should remove it in v2

licui3936 commented on docs.README.md:
```diff
@@ -1,82 +1,241 @@
+* {@link Application#scheduleRestart scheduleRestart} _Application_
+* {@link InterApplicationBus#send send} _InterApplicationBus_
+* {@link Notification#sendMessage sendMessage} _Notification_
+* {@link Notification#sendMessageToApplication sendMessageToApplication} _Notification_
```
should remove it in v2

licui3936 commented on docs.README.md:
```diff
@@ -1,82 +1,241 @@
+* {@link ExternalApplication#removeEventListener removeEventListener} _ExternalApplication_
+* {@link Frame#removeEventListener removeEventListener} _Frame_
+* {@link System#removeEventListener removeEventListener} _System_
+* {@link Window#removeEventListener removeEventListener} _Window_
```
all 'removeEventListner' should be removed in v2.
